common/src/main/java/com/wynntils/models/lootrun/scoreboard/LootrunScoreboardPart.java

```diff
@@ -56,6 +58,11 @@ public void onSegmentChange(ScoreboardSegment newValue) {
         } else if (currentStateLine.matches(DEFEND_PATTERN, PartStyle.StyleType.NONE)) {
             Models.Lootrun.setState(LootrunningState.IN_TASK, LootrunTaskType.DEFEND);
         }
+
+        currentStateLine = content.get(1);
+        if (currentStateLine.matches(TIMER_PATTERN, PartStyle.StyleType.NONE)) {
```

Comment:
Then down here, if you make a Matcher object like `Matcher timeMatcher = currentStateLine.getMatcher(TIMER_PATTERN);` and if that matches, you can use `timeMatcher.group(1)` to get the minutes